rasa_core/train.py

```diff
@@ -332,6 +339,12 @@ def do_interactive_learning(cmdline_args, stories, additional_arguments):
         finetune=cmdline_args.finetune,
         skip_visualization=cmdline_args.skip_visualization)
 
+def validate_files(domain, stories, intents):
+    validator = Validator(domain=domain,
```

Comment:
(Seeing my other comment, this would now be `validator = Validator.from_path(......)`